packages/sidenav/src/components/SideBar/SideBar.styled.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from "prop-types";
+import styled, { css } from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+import { FULLSCREEN, PUSH, OVERLAY } from "../../constants/variants";
+
+const fullScreenStyle = css`
+  top: 0;
+  padding: ${({ expanded }) => (expanded ? "16px" : "16px 0")};
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  
+  ${
+    "" /* keep 1px width when collapsed to prevent safari from lose backdrop-filter setting it to 0 */
+  }
```

Comment:
We don't need to put the comment inside a placeholder. CSS accepts comments.

Also, a little grammar fix suggestion:
```css
  /* Keeping 1px width when collapsed to prevent Safari from losing 
     backdrop-filter if it's set to 0 */
```